### PR TITLE
Added qTox download for OS X

### PR DESCRIPTION
**Pull request by nurupo**
Review?

@zetok @optimumtact

### CODE REVIEW

**Comment by nurupo** [comment]
Forgot that we have to add `style="margin-top:-15px;"`.

File: themes/website/templates/download.html [hunk 1]
```diff
@@ -44,7 +44,8 @@ <h1 class="header">Thanks for downloading!</h1>
 				<div class="col-sm-4 feature">
 					<img src="theme/img/plat/mac_dark.svg">
 					<h2>OS X</h2>
-					<p class="lead"><a href="https://github.com/uTox/uTox/releases/download/v0.9.8/uTox-0.9.8.dmg">uTox 64 bit (OS X 10.7+)</a></p>
+                                        <p class="lead"><a href="https://github.com/qTox/qTox/releases/download/v1.5.1/qTox.dmg">qTox</a></p>
```
Comment (nurupo) [inline themes/website/templates/download.html]: Oh, I have used spaces instead of tabs. Will fix that.

**Comment by nurupo** [comment]
@stal888 good?

**Comment by stal888** [comment]
Aye-aye

**Pull request merged**